src/components/customSwitch/CustomSwitch.tsx

```diff
@@ -0,0 +1,25 @@
+interface CustomSwitchProps {
+  checked: boolean;
+  onChange: () => void;
+}
+
+const CustomSwitch = (props: CustomSwitchProps) => {
+  const { checked, onChange } = props;
+
+  return (
+    <label className="inline-flex items-center cursor-pointer">
+      <input
+        type="checkbox"
+        value=""
+        className="sr-only peer"
+        onChange={onChange}
+        readOnly
+        checked={checked}
+      />
+      <div className="relative w-11 h-6 bg-gray-200 peer-focus:outline-none peer-focus:ring-4 peer-focus:ring-green-300 dark:peer-focus:ring-green-800 rounded-full peer dark:bg-gray-700 peer-checked:after:translate-x-full rtl:peer-checked:after:-translate-x-full peer-checked:after:border-white after:content-[''] after:absolute after:top-[2px] after:start-[2px] after:bg-white after:border-gray-300 after:border after:rounded-full after:h-5 after:w-5 after:transition-all dark:border-gray-600 peer-checked:bg-green-600"></div>
```

Comment:
Feels like there should be a shorter way to do this but maybe there's not

Reply:
It has to be that long 😸